CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ option(ENABLE_KERNEL_ADDRESS_SANITIZER "Enable kernel address sanitizer testing
 option(ENABLE_MEMORY_SANITIZER "Enable memory sanitizer testing in gcc/clang" OFF)
 option(ENABLE_UNDEFINED_SANITIZER "Enable undefined behavior sanitizer testing in gcc/clang" OFF)
 option(ENABLE_FUZZER_SANITIZER "Enable fuzzer sanitizer testing in clang" OFF)
+option(ENABLE_EXTRA_KERNEL_DEBUG_SYMBOLS "Set -Og and -ggdb3 options for Kernel code for easier debugging" OFF)
 option(ENABLE_ALL_THE_DEBUG_MACROS "Enable all debug macros to validate they still compile" OFF)
```

Comment:
Maybe we could have something like `ENABLE_ALL_DEBUG_FACILITIES` that we can set in CI and we could enable all this extra stuff under it, instead of hijacking the meaning of `ENABLE_ALL_DEBUG_MACROS`?

Reply:
Shore, added that, and replaced in CI